tasks/data_handling/data_pre_processing.py

```diff
@@ -0,0 +1,149 @@
+import scanpy as sc
+
+def create_count_matrix(file_path : str = None) -> sc.AnnData:
+    """
+    ### Function that transforms h5-file to a count matrix.
+
+    ---
+    
+    #### Args:
+        - file_path (str): Path to a 10x hdf5 file.
+
+    #### Returns: 
+        - AnnData: Count matrix from h5-file in file_path with unique variables
+
+    ---
+
+    Annotated data matrix, where observations/cells are named by their barcode and variables/genes by gene name. Stores the following information:
+
+    `~anndata.AnnData.X`
+        The data matrix is stored
+    `~anndata.AnnData.obs_names`
+        Cell names
+    `~anndata.AnnData.var_names`
+        Gene names
+    `~anndata.AnnData.var`\ `['gene_ids']`
+        Gene IDs
+    `~anndata.AnnData.var`\ `['feature_types']`
+        Feature types
+    ---
+
+    Written: ronjah@chalmers.se
+    """
+    if file_path == None:
+        raise ValueError('Need a path to h5-file.')
+    adata = sc.read_10x_h5(file_path)
```

Comment:
Why 'read_10x' and not 'read'?

Reply:
Since that function is specifically made for 10x file, which is the format we have and will get if I remember correctly? I suppose read could work as well but it is more general.

Reply:
I understand. Please write in the name of the function (and its description) something to indicate this. Namely, that we are expecting a 10x file